pycoin/services/providers.py

```diff
@@ -5,7 +5,7 @@
 from .tx_db import TxDb
 
 
-SERVICE_PROVIDERS = ["BLOCKCHAIN_INFO", "BLOCKEXPLORER", "BLOCKR_IO", "BITEASY"]
+SERVICE_PROVIDERS = ["BLOCKCHAIN_INFO", "BLOCKEXPLORER", "BLOCKR_IO", "BITEASY", "CHAIN_COM"]
 
```

Comment:
I don't want to add CHAIN_COM to this list because I haven't yet figured out how to pass AUTH info in environment variables.